PantheonCMD/pcmd.py

```diff
@@ -189,6 +189,13 @@ def parse_args():
     # Else parse actions
     # Action - preview
     if args.command == 'preview':
+      
+        output_format = 'html'
+
+        if args.format:
+            if args.format == 'pdf':
+                output_format = 'pdf'
```

Comment:
Can this work? Just wanna get rid of the nested if statement 
```suggestion
        if args.format == 'pdf':
                output_format = 'pdf'
        else:
                output_format = 'html'
```